Modules/Devices/test/Devices.Domain.Tests/Identities/RejectDeletionProcessTests.cs

```diff
@@ -8,49 +8,66 @@
 using Xunit;
 
 namespace Backbone.Modules.Devices.Domain.Tests.Identities;
-public class RejectDeletionProcessTest
+public class RejectDeletionProcessTests
 {
     [Fact]
     public void Reject_deletion_process_waiting_for_approval()
     {
         // Arrange
-        SystemTime.Set(SystemTime.UtcNow);
+        SystemTime.Set(DateTime.Parse("2020-01-01"));
         var identity = CreateIdentityWithDeletionProcessWaitingForApproval();
-        var deviceId = DeviceId.Parse("DVC"); // temporary solution until feature-cancellation-of-a... branch is merged
+        identity.Devices.Add(new Device(identity));
+        var deviceId = identity.Devices[0].Id;
 
         // Act
-        identity.RejectDeletionProcess(identity.GetDeletionProcessInStatus(DeletionProcessStatus.WaitingForApproval)!.Id, deviceId);
+        identity.RejectDeletionProcess(identity.DeletionProcesses[0].Id, deviceId);
 
         // Assert
-        identity.Status.Should().Be(IdentityStatus.Active);
-
         var deletionProcess = identity.DeletionProcesses.FirstOrDefault(d => d.Status == DeletionProcessStatus.Rejected)!;
         AssertAuditLogEntryWasCreated(deletionProcess);
     }
 
+    [Fact]
+    public void Throws_when_device_not_owned_by_identity()
+    {
+        // Arrange
+        SystemTime.Set(DateTime.Parse("2020-01-01"));
```

Comment:
Why do you set SystemTime?

Reply:
Oversight, I failed to remove unnecessary parts of copied code.